isis/scripts/isisVarInit.py

```diff
@@ -20,12 +22,50 @@
 #
 # SPDX-License-Identifier: CC0-1.0
 
+# This is the full path to this environment:
+CONDA_PREFIX = os.environ["CONDA_PREFIX"]
```

Comment:
If no conda env is activated this will throw right? Should we have a os.environ.get('CONDA_PREFIX', None) and then throw an error?